docs/user-guide/binned-data/computation.ipynb

```diff
@@ -13,8 +13,40 @@
     "\n",
     "1. [Bin-centric arithmetic](#Bin-centric-arithmetic) treats every bin as an element to, e.g., apply a different scale factor to every bin.\n",
     "2. [Event-centric arithmetic](#Event-centric-arithmetic) considers the individual events within bins.\n",
-    "   This allows for operation without the precision loss that would ensue from simply histogramming data.\n",
+    "   This allows for operation without the precision loss that would ensue from simply histogramming data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview and Quick Reference\n",
+    "\n",
+    "Before going into a detailed explanation below we provide a quick reference:\n",
+    "\n",
+    "- Unary operations such as `sin`, `cos`, or `sqrt` work as normal.\n",
+    "- Comparison operations such as `less` (`<`) are not supported.\n",
+    "- Binary operations such as `+` work in principle, but usually not if both operands represent event data.\n",
+    "  In that case, see table below.\n",
     "\n",
+    "Given two data arrays `a` and `b`, equivalent operations are:\n",
+    "\n",
+    "Dense data operation | Binned data equivalent | Comment\n",
+    ":--- |:--- |:---\n",
+    "`a + b` | `a.bins.concatenate(b)` | if both `a` and `b` are event data\n",
+    "`a - b` | `a.bins.concatenate(-b)` | if both `a` and `b` are event data\n",
+    "`a += b` | `a.bins.concatenate(b, out=a)` | if both `a` and `b` are event data\n",
+    "`a -= b` | `a.bins.concatenate(-b, out=a)` | if both `a` and `b` are event data\n",
+    "`sc.sum(a, 'dim')` | `a.bins.concatenate('dim')` | \n",
+    "`sc.mean(a, 'dim')` | not available | `min`, `max`, and other similar reductions are also not available\n",
+    "`sc.rebin(a, dim, 'edges')` | `sc.bin(a, edges=[edges])` | \n",
+    "`groupby(...).sum('dim')` | `groupby(...).bins.concatenate('dim')` | `mean`, `max`, and other similar reductions are also available"
```

Comment:
These comparative of operations for dense binned are nice additions. Either here or separately, we should also update `from_mantid_to_scipp` to include the differences of these equivalent operations.

Reply:
Agree, but separately, maybe even after the release, which I would prioritze getting out now.